### PR TITLE
Add new strings and translate to PT-BR

### PR DESCRIPTION
**Pull request by bitcoinuser**

### CODE REVIEW

File: src/locales/pt_br/translation.json [hunk 1]
```diff
@@ -763,6 +769,29 @@
...
+        "myProfit": "Lucro",
+        "rewards": "Recompensas"
+      }
+    },
+    "modal": {
+      "deposit": {
+        "title": "Emprestar",
+        "cta": "Depositar",
+        "amount": "Quantidade:",
+        "apy": "APY% <0/> variável"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:786]:
```suggestion
        "apy": "Juros anual% <0/> variável"
```

File: src/locales/pt_br/translation.json [hunk 2]
```diff
@@ -1057,13 +1113,13 @@
       "title": "Conectar",
       "line1": "Use uma das carteiras disponíveis para conectar na rede <0>RSK</0>",
       "line2": "Ainda não tem uma carteira? Recomendamos a <0>carteira Liquality</0>",
-      "cta": "Conectar carteira"
+      "cta": "Conectar Carteira"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1116]:
```suggestion
      "cta": "Conectar carteira"
```

File: src/locales/pt_br/translation.json [hunk 3]
```diff
@@ -1057,13 +1113,13 @@
...
     },
     "topUp": {
       "title": "Adquirir <0>RBTC</0>",
       "line1": "Você irá precisar de <0>RBTC</0> na sua carteira para comprar SOV e pagar as taxas da rede RSK",
       "line2": "Saiba mais sobre <0>RBTC</0> na RSK",
-      "cta": "Carregar carteira"
+      "cta": "Carregar Carteira"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1122]:
```suggestion
      "cta": "Carregar carteira"
```

File: src/locales/pt_br/translation.json [hunk 4]
```diff
@@ -738,6 +766,10 @@
     "emptyState": "Histórico vázio.",
     "walletHistory": "A carteira precisa estar conectada.",
     "loading": "Carregando..."
+  },
+   "validationErrors": {
+    "insufficientBalance": "Sem saldo suficiente",
+    "minimumZero": "Insira uma quantidade maior que 0"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:772]:
```suggestion
    "minimumZero": "Insira uma quantia maior que 0"
```

File: src/locales/pt_br/translation.json [hunk 5]
```diff
@@ -738,6 +766,10 @@
     "emptyState": "Histórico vázio.",
     "walletHistory": "A carteira precisa estar conectada.",
     "loading": "Carregando..."
+  },
+   "validationErrors": {
+    "insufficientBalance": "Sem saldo suficiente",
+    "minimumZero": "Insira uma quantia maior que 0"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:772]:
```suggestion
    "minimumZero": "Insira uma quantidade maior que 0"
```

File: src/locales/pt_br/translation.json [hunk 6]
```diff
@@ -226,7 +226,33 @@
...
+        "startPrice": "Preço inicial"
+      },
+      "inputLabel": "Insira um endereço de e-mail",
+      "submitButton": "Inscreva-se para receber atualizações"
+    },
+    "previousSales": {
+      "title": "Vendas anteriores",
+      "projectCard": {
+        "saleName": "Nome",
+        "saleAllocation": "Reservado para venda",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:248]:
```suggestion
        "saleAllocation": "Disponibilizado para venda",
```

File: src/locales/pt_br/translation.json [hunk 7]
```diff
@@ -735,9 +763,13 @@
       "transferTx": "ID recebimento RBTC ",
       "txHash": "Transação"
     },
-    "emptyState": "Histórico vázio.",
+    "emptyState": "Você não tem histórico de depósitos.",
     "walletHistory": "A carteira precisa estar conectada.",
     "loading": "Carregando..."
+  },
+   "validationErrors": {
+    "insufficientBalance": "Sem saldo suficiente",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:771]:
```suggestion
    "insufficientBalance": "Saldo insuficiente",
```

File: src/locales/pt_br/translation.json [hunk 8]
```diff
@@ -280,7 +306,7 @@
       "allVolumne": "TODO PERÍODO",
       "btc": "BTC",
       "usd": "USD",
-      "tvl": "Valor total na Sovryn",
+      "tvl": "TVL",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:309]:
```suggestion
      "tvl": "TVL (Valor total na Sovryn)",
```

File: src/locales/pt_br/translation.json [hunk 9]
```diff
@@ -667,7 +694,7 @@
       "description": "Informações sobre a carteira conectada"
     },
     "tabs": {
-      "userAssets": "Meus criptoativos",
+      "userAssets": "Criptoativos",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:697]:
```suggestion
      "userAssets": "Meus criptoativos",
```

File: src/locales/pt_br/translation.json [hunk 10]
```diff
@@ -763,6 +795,29 @@
         "line_2": "O pool atual tem somente {{amount}} {{currency}} disponível.",
         "line_3": "O pool irá aumentar quando os usuários devolverem seus respectivos empréstimos."
       }
+    },
+    "rowTable": {
+      "tableHeaders": {
+        "apy": "Juros anual%",
+        "yourDeposit": "Meu depósito",
+        "myProfit": "Lucro",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:803]:
```suggestion
        "myProfit": "Rendimento",
```

File: src/locales/pt_br/translation.json [hunk 11]
```diff
@@ -763,6 +795,29 @@
         "line_2": "O pool atual tem somente {{amount}} {{currency}} disponível.",
         "line_3": "O pool irá aumentar quando os usuários devolverem seus respectivos empréstimos."
       }
+    },
+    "rowTable": {
+      "tableHeaders": {
+        "apy": "Juros anual%",
+        "yourDeposit": "Meu depósito",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:802]:
```suggestion
        "yourDeposit": "Você emprestou",
```

File: src/locales/pt_br/translation.json [hunk 12]
```diff
@@ -763,6 +795,29 @@
...
+        "cta": "Depositar",
+        "amount": "Quantidade:",
+        "apy": "Juros anual% <0/> variável"
+      },
+      "withdraw": {
+        "title": "Resgatar empréstimo",
+        "cta": "Resgatar",
+        "amount": "Quantidade:",
+        "reward": "Recompensa:",
+        "depositBalance": "Meu depósito: <0/> <1/>"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:819]:
```suggestion
        "depositBalance": "Você emprestou: <0/> <1/>"
```

File: src/locales/pt_br/translation.json [hunk 13]
```diff
@@ -280,7 +306,7 @@
       "allVolumne": "TODO PERÍODO",
       "btc": "BTC",
       "usd": "USD",
-      "tvl": "Valor total na Sovryn",
+      "tvl": "TVL (Valor total na Sovryn)",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:309]:
```suggestion
      "tvl": "TVL",
```

File: src/locales/pt_br/translation.json [hunk 14]
```diff
@@ -218,15 +218,41 @@
     "finance": "Finanças",
     "lend": "Emprestar criptos",
     "borrow": "Obter empréstimos",
-    "liquidity": "Minerar liquidez",
+    "yieldFarm": "Farmar",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:221]:
```suggestion
    "yieldFarm": "Farmar rendimentos",
```

File: src/locales/pt_br/translation.json [hunk 15]
```diff
@@ -988,6 +1044,8 @@
     }
   },
   "liquidityMining": {
+    "deposit": "Adicionar",
+    "withdraw": "Retirar",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1048]:
```suggestion
    "withdraw": "Resgatar",
```

File: src/locales/pt_br/translation.json [hunk 16]
```diff
@@ -988,6 +1044,8 @@
     }
   },
   "liquidityMining": {
+    "deposit": "Adicionar",
+    "withdraw": "Resgatar",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1048]:
```suggestion
    "withdraw": "Retirar",
```

**Review by creed-victor — DISMISSED** [review]

**Review by pietro-maximoff — APPROVED** [review]

File: src/locales/pt_br/translation.json [hunk 17]
```diff
@@ -211,22 +211,49 @@
...
-    "buySov": "Compre SOV",
+    "buySov": "Comprar SOV",
     "trade": "Negociar",
     "swap": "Trocar criptos",
     "marginTrade": "Operar margem",
     "finance": "Finanças",
     "lend": "Emprestar criptos",
     "borrow": "Obter empréstimos",
-    "liquidity": "Minerar liquidez",
+    "yieldFarm": "Farmar rendimentos",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:221]:
```suggestion
    "yieldFarm": "Minerar liquidez",
```

File: src/locales/pt_br/translation.json [hunk 18]
```diff
@@ -1105,15 +1164,20 @@
...
+        "text": "Na primeira semana, $1.2 milhões de SOV serão distribuídos para os fornecedores de liquidez do AMM. Esta super promoção irá durar 7 dias. Algumas análises indicam que os primeiros participantes terão um retorno 3000%!! O maior rendimento de ETH existente hoje. Haverá novas recompensas para as próximas semanas, que serão revisadas e anunciadas no final de cada semana.",
+        "cta": "Ganhe agora"
       }
     },
     "features": {
       "title": "Como você pode usar a Sovryn",
       "learnMore": "Saiba mais",
       "mining": {
-        "title": "Minerando liquidez",
+        "title": "Farmando rendimentos",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1178]:
```suggestion
        "title": "Minerando liquidez",
```

File: src/locales/pt_br/translation.json [hunk 19]
```diff
@@ -1105,15 +1164,20 @@
...
         "text": "Diga adeus aos serviços centralizados de fornecimento de liquidez. Na Sovryn, os provedores de liquidez são recompensados por facilitar as negociações e trocas de criptoativos.",
-        "cta": "Minerar"
+        "cta": "Farmar"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1180]:
```suggestion
        "cta": "Minerar"
```

**Review by pietro-maximoff — APPROVED** [review]

**Review by creed-victor — APPROVED** [review]

**Comment by bitcoinuser** [comment]
Hi @creed-victor this should be merged on master. Now on UI I have buttons in English mixed with Portuguese. Please merge this on Master. Thanks

**Pull request merged**